content/02.introduction.md

```diff
@@ -25,16 +26,28 @@ Further, we aim to amplify the true signal out of the vast noise produced by tho
+#### Coronavirus Genome and Structure
+
+Coronaviruses (CoVs; order *Nidovirales*, family *Coronaviridae*, subfamily *Orthocoronavirinae*) are enveloped viruses with some of the largest viral RNA genomes.
+Coronavirus genomes range from 27 to 32 kilobases (Kb) in length.
+The SARS-CoV-2 genome lies in the middle of this range at 29,903 bp [@doi:10/ggjr43].
+There are several fundamental genomic characteristics shared by all viruses in the *Nidovirales* order, including enveloped, non-segmented, and positive-sense (ssRNA+) genomes and a large number of non-structural genes originating through ribosomal frameshifting [@doi:10.1007/978-1-4939-2438-7_1].
+Genome organization is highly conserved within the order [@doi:10.1007%2F978-1-4939-2438-7_1].
+Coronavirus genomes are comprised of a replicase gene that is two-thirds of the genome.
+This polypeptide is translated into 16 non-structural proteins (nsp1-16, except in Gammacoronaviruses, where nsp1 is absent), which form replication machinery used to synthesize viral RNA [@doi:10.1002/jmv.25681].
+The remaining third of the genome encodes structural proteins, including the spike (S), membrane (M), envelope (E), and nucleocapsid (N) proteins.
+Additional accessory genes are sometimes present between these two regions, depending on the species or strain.
+<!-- To do: Add a figure showing the structure of the genome.-->
+<!-- Also, everything focuses on S! Should we also talk about others?-->
 
-Coronaviruses (CoVs; order *Nidovirales*, family *Coronaviridae*, subfamily *Orthocoronavirinae*) are enveloped viruses with some of the largest viral RNA genomes and helical capsids with the solar corona-like appearance [@isbn:0781760607].
-All viruses in the *Nidovirales* order are characterized by: 1) enveloped, non-segmented, positive-sense RNA genomes with a highly conserved genomic organization; 2) many non-structural genes by ribosomal shifting; 3) several unusual enzymatic activities encoded within the replicase-transcriptase polyprotein; and 4) 3' nested sub-genomic mRNAs [@doi:10.1007/978-1-4939-2438-7_1].
 Coronaviruses are classified into four genera: alpha, beta, delta and gamma coronaviruses.
 Among them, alpha and beta coronaviruses infect mammalian species, gamma coronaviruses infect avian species and delta coronaviruses infect both mammalian and avian species [@doi:10.1146/annurev-virology-110615-042301].
 The viruses were initially subdivided into these genera based on antigenic relationships of the spike (S), membrane (M), envelope (E), and nucleocapsid (N) proteins but are now divided by phylogenetic clustering.
+[this section may be better as a figure panel + legend]
+Phylogenetic analysis of a PCR amplicon fragment from five patients along with the total virus genome of 29.8 kilobases indicates that the virus is a novel betacoronavirus belonging to the B lineage, also known as sarbecovirus.
+The sarbecovirus lineage also includes the human SARS coronavirus [@doi:10/ggjs7j].
```

Comment:
I cannot seem to find this doi...

Reply:
It's a short DOI -- manubot doesn't like parentheses in citations but The Lancet loves them :) If you see something like that again, you can use doi.org/10/ggjs7j -- but [here is the link](https://www.thelancet.com/pdfs/journals/lancet/PIIS0140-6736(20)30154-9.pdf) to this paper! They are generated by shortdoi.org, in case you ever need to know how to do it!